bosh/releases/generators/pre_render_scripts/generator.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+set -o errexit -o nounset
+
+touch "${OUTPUT}"
+
+function append {
+  echo "${1}" >> "${OUTPUT}"
+}
+
+append "- type: replace"
+append "  path: /instance_groups/name=${INSTANCE_GROUP}/jobs/name=${JOB}/properties/bosh_containerization?/pre_render_scripts/-"
+append "  value: |"
+append "$(cat "${PRE_RENDER_SCRIPT}" | sed 's/^/    /')"
```

Comment:
How about  `cat "${PRE_RENDER_SCRIPT}" | sed 's/^/    /' >> "${OUTPUT}"` instead ?
The current construction pulls the entire pre-render-script into memory before writing it out again.
The proposed construction avoids that, streaming it directly from input to output without taking memory.

Reply:
I should say that this is a nit.

Reply:
I don't think the size of the script is significant. In any case, it's always good to have it done in the best way. In this case, it doesn't sacrifice readability, so I used your suggestion. I also simplified the script removing the function. Thanks for the suggestion.